Modules/itertoolsmodule.c

```diff
@@ -3454,12 +3454,81 @@ count_repr(countobject *lz)
                                 lz->long_cnt, lz->long_step);
 }
 
+static PyObject *
+count_peek(countobject *lz)
```

Comment:
```suggestion
count_peek(countobject *lz, PyObject *Py_UNUSED(ignored))
```

The function must have 2 parameters.  This should fix warnings from gh actions below.